Pipfile

```diff
@@ -7,23 +7,24 @@ verify_ssl = true
 flake8 = "*"
 
 [packages]
-uvloop = "==0.15.3"
-aiohttp = "==3.7.4"
-pytest = "==6.2.4"
-aiofiles = "==0.3.1"
-pytest-aiohttp = "==0.1.3"
-coverage = "==4.4.2"
-pytest-cov = "==2.5.1"
-hypothesis = "==6.14.5"
-globus-sdk = "==1.6.1"
-python-dotenv = "==0.14.0"
+aiofiles = "==23.1.0"
+aiohttp = "==3.8.4"
 aiohttp_cors = "==0.7.0"
-frozendict = "==2.0.3"
-pandas = "==1.3.2"
-xlrd = "==2.0.1"
-openpyxl = "==3.0.7"
+black = "==23.7.0"
```

Comment:
Added black, wasn't here before.

Reply:
might be easier to use ruff -- linting and code formatting in one beautiful package!

Reply:
We have ruff already here  as of https://github.com/kbase/staging_service/pull/196

Reply:
I'm happy to reorganize that, too, if we like. I'll get it in a separate PR. I just wanted the Pipfile and requirements up to date with each other (more or less).